packages/sveltekit/src/supabaseLoadClient.ts

```diff
@@ -82,6 +82,9 @@ export function createSupabaseLoadClient<
   cookieOptions?: CookieOptions;
   /**
    * The event listener only runs in the browser.
+   * @deprecated DO NOT USE THIS
+   *
+   * use this instead: https://supabase.com/docs/guides/auth/auth-helpers/sveltekit#setting-up-the-event-listener-on-the-client-side
```

Comment:
We should probably inline the content from the link. So tell the user rather than redirecting them. We can add the link below that.